Description: Fix code to expect translation files in the standard folder
Author: Margarita Manterola <marga@debian.org>
Origin: vendor
Forwarded: no
Applied-Upstream: yes
Last-Update: 2014-07-12
Index: cinnamon-session/capplet/main.c
===================================================================
--- cinnamon-session.orig/capplet/main.c	2014-05-24 15:43:03.533017516 +0200
+++ cinnamon-session/capplet/main.c	2014-07-12 14:28:49.257009695 +0200
@@ -79,7 +79,7 @@
         GError    *error;
         GtkWidget *dialog;
 
-        bindtextdomain ("cinnamon-session", "/usr/share/cinnamon/locale");
+        bindtextdomain ("cinnamon-session", "/usr/share/locale");
         bind_textdomain_codeset ("cinnamon-session", "UTF-8");
         textdomain ("cinnamon-session");
 
